src/doc/trpl/installing-rust.md

```diff
@@ -26,48 +26,108 @@ the [64-bit installer][win64] and run it.
 [win32]: https://static.rust-lang.org/dist/rust-1.0.0-beta-i686-pc-windows-gnu.msi
 [win64]: https://static.rust-lang.org/dist/rust-1.0.0-beta-x86_64-pc-windows-gnu.msi
 
+If you would like to build from source, please check out the
+documentation on [building Rust from Source][from source], there is always [the official
+binary downloads][install page] as well. 
+
+[from source]: https://github.com/rust-lang/rust#building-from-source
+[install page]: http://www.rust-lang.org/install.html
+
+If you're on Linux or Mac rust has a tool called ['multirust'](https://github.com/brson/multirust/README.md) for managing
```

Comment:
I love multirust too, but if it's not production ready yet, I want to give it time to bake first, you know?